recipes/trompeloeil/config.yml

```diff
@@ -0,0 +1,11 @@
+versions:
+  35:
+    folder: all
+  36:
+    folder: all
+  37:
+    folder: all
+  38:
+    folder: all
+  39:
+    folder: all
```

Comment:
only keep the version 39?

Reply:
Yes, as requested by @Croydon : https://github.com/conan-io/conan-center-index/pull/3901#discussion_r543461105

Reply:
done